client/src/app/site/pages/meetings/pages/participants/export/speaker-csv-export.service/speaker-csv-export.service.ts

```diff
@@ -14,20 +21,45 @@ export const speakerHeadersAndVerboseNames = {
     structureLevelName: _(`Structure level`),
     user_number: _(`Participant number`),
     speech_state: _(`Speech type`),
-    speakingTime: _(`Time`),
+    begin_time: _(`Start time`),
+    speakingTime: _(`Duration`),
     topic: _(`Element`)
 };
 
 @Injectable({
     providedIn: ParticipantExportModule
 })
 export class SpeakerCsvExportService {
-    public constructor(private csvExport: MeetingCsvExportForBackendService, private translate: TranslateService) {}
+    public constructor(
+        private csvExport: MeetingCsvExportForBackendService,
+        private translate: TranslateService,
+        private durationService: DurationService,
+        private dfnsConfig: DateFnsConfigurationService
+    ) {}
+
+    private col_begin_time(speaker: ViewSpeaker): string {
+        return speaker.begin_time
+            ? format(fromUnixTime(speaker.begin_time), `Ppp`, {
+                  locale: this.dfnsConfig.locale()
+              })
+            : ``;
+    }
+
+    private col_speakingTime(speaker: ViewSpeaker): string {
+        return this.durationService.durationToString(speaker.speakingTime, ``);
+    }
 
     public export(speakers: ViewSpeaker[]): void {
         this.csvExport.export(
             speakers,
             Object.keys(speakerHeadersAndVerboseNames).map(key => {
+                if (this[`col_` + key]) {
```

Comment:
This would be a bit dangerous if a subsequent programmer doesn't see the connection with the `col_` functions.
Please do this in another way